site/css/typography/_fonts.css

```diff
@@ -0,0 +1,88 @@
+/**
+* Licensing fonts from 2 independent providers
+*/
+
+/**
+ * @license
+ * MyFonts Webfont Build ID 3286567, 2016-09-22T07:01:50-0400
+ *
+ * The fonts listed in this notice are subject to the End User License
+ * Agreement(s) entered into by the website owner. All other parties are
+ * explicitly restricted from using the Licensed Webfonts(s).
+ *
+ * You may obtain a valid license at the URLs below.
+ *
+ * Webfont: ProximaNova-Bold by Mark Simonson
+ * URL: http://www.myfonts.com/fonts/marksimonson/proxima-nova/bold/
+ *
+ * Webfont: ProximaNova-Light by Mark Simonson
+ * URL: http://www.myfonts.com/fonts/marksimonson/proxima-nova/light/
+ *
+ *
+ * License: http://www.myfonts.com/viewlicense?type=web&buildid=3286567
+ * Licensed pageviews: 1,000,000
+ * Webfonts copyright: Copyright (c) Mark Simonson, 2005. All rights reserved.
+ *
+ * © 2016 MyFonts Inc
+*/
+
+/*
+  Copyright (C) 2011-2016 Hoefler & Co.
+  This software is the property of Hoefler & Co. (H&Co).
+  Your right to access and use this software is subject to the
+  applicable License Agreement, or Terms of Service, that exists
+  between you and H&Co. If no such agreement exists, you may not
+  access or use this software for any purpose.
+  This software may only be hosted at the locations specified in
+  the applicable License Agreement or Terms of Service, and only
+  for the purposes expressly set forth therein. You may not copy,
+  modify, convert, create derivative works from or distribute this
+  software in any way, or make it accessible to any third party,
+  without first obtaining the written permission of H&Co.
+  For more information, please visit us at http://typography.com.
+  296519-110129-20160915
+*/
+
+/* Impressions counter required by the license */
+@import url("//hello.myfonts.net/count/322627");
+
+@font-face {
+    font-family: 'ProximaNova-Bold';
+    src: url('./fonts/322627_0_0.woff2') format('woff2'), url('./fonts/322627_0_0.woff') format('woff'), url('./fonts/322627_0_0.ttf') format('truetype');
+}
+
+@font-face {
+    font-family: 'ProximaNova-Light';
+    src: url('./fonts/322627_1_0.woff2') format('woff2'), url('./fonts/322627_1_0.woff') format('woff'), url('./fonts/322627_1_0.ttf') format('truetype');
+}
+
+/* Importing base64 encoded Sentinel fonts */
+@import url("./fonts/sentinel-woff.css");
+@import url("./fonts/sentinel-woff2.css");
+@import url("./fonts/sentinel-opentype.css");
+@import url("./fonts/sentinel-truetype.css");
+
+.fontXL {
```

Comment:
That doesn't work with css modules anymore since hyphen is an operator in Javascript